### PR TITLE
Improve accessibility

### PR DESCRIPTION
**Pull request by schlan**
### Changes
- Always expand image picker if accessibility services are enabled
- Add missing labels and content descriptions
- Add yml for translations

### Reviewers
@brendan-fahy @baz8080 @schlan @eepDev

### References
- None

### Risks
- None

### CODE REVIEW

File: MobileSDKBelvedere.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+parts:
+# Dialog
+  - translation:
+      key: "bsdk_belvedere_dialog_gallery"
+      title: "Label of a button that allows the user to select an image from the image gallery"
+      value: "Gallery"
+  - translation:
+      key: "bsdk_belvedere_dialog_camera"
+      title: "Label of a button that allows the user to select an image from the image gallery"
+      value: "No connection"
```
Comment (brendan-fahy) [inline MobileSDKBelvedere.yml]: The key, title, and value here don't seem to match. Is this correct?

File: belvedere/src/main/java/zendesk/belvedere/BelvedereDialog.java [hunk 2]
```diff
@@ -232,7 +232,7 @@ public static AttachmentSource from(MediaIntent belvedereIntent, Context context
             } else if (belvedereIntent.getTarget() == MediaIntent.TARGET_DOCUMENT) {
                 return new AttachmentSource(R.drawable.belvedere_ic_image, context.getString(R.string.belvedere_dialog_gallery));
             } else {
-                return new AttachmentSource(-1, context.getString(R.string.belvedere_dialog_unknown));
+                return new AttachmentSource(-1, "");
```
Comment (brendan-fahy) [inline belvedere/src/main/java/zendesk/belvedere/BelvedereDialog.java]: ?

File: MobileSDKBelvedere.yml (hunk 1 above)
Comment (schlan) [reply MobileSDKBelvedere.yml]: Fixed.

File: belvedere/src/main/java/zendesk/belvedere/BelvedereDialog.java (hunk 2 above)
Comment (schlan) [reply belvedere/src/main/java/zendesk/belvedere/BelvedereDialog.java]: This was an edge case in the deprecated dialog from v1, we don't need a translated string here.

**Comment by schlan** [comment]
cc @zendesk/i18n @zendesk/i18n-dev

**Pull request merged**